### PR TITLE
Implement Automatic Migrations

### LINKED ISSUE #22: Run migrations automatically as part of deployment
Set up deployment workflow to also handle running migrations manually as part of deployment.

### PR DESCRIPTION
**Pull request by seun-beta**
fixes #22

### CODE REVIEW

File: migrations/versions/6bbae2fceac4_.py [hunk 1]
```diff
@@ -1,68 +0,0 @@
...
-    sa.Column('book_id', postgresql.UUID(), autoincrement=False, nullable=False),
-    sa.Column('borrower', postgresql.UUID(), autoincrement=False, nullable=False),
-    sa.Column('created_at', postgresql.TIMESTAMP(), autoincrement=False, nullable=False),
-    sa.Column('updated_at', postgresql.TIMESTAMP(), autoincrement=False, nullable=True),
-    sa.ForeignKeyConstraint(['book_id'], ['book.id'], name='borrowing_book_id_fkey'),
-    sa.ForeignKeyConstraint(['borrower'], ['user_profile.id'], name='borrowing_borrower_fkey'),
-    sa.PrimaryKeyConstraint('id', name='borrowing_pkey')
-    )
-    op.drop_table('borrow')
-    # ### end Alembic commands ###
```
Comment (kosyfrances) [inline migrations/versions/6bbae2fceac4_.py:68]: Why delete this? 🤔

File: .github/workflows/python-app.yml [hunk 2]
```diff
@@ -36,6 +36,26 @@ jobs:
...
         
+          
+    - name: Test using unittest
+      run: |
+        python -m unittest
+      env:
+        TEST_TOKEN: ${{ secrets.TEST_TOKEN }}
+        DATABASE_URL: "${{ secrets.DATABASE_URI}}"
+        EMAIL: ${{ secrets.EMAIL }}
+        SECRET_KEY: ${{ secrets.SECRET_KEY }}
```
Comment (kosyfrances) [inline .github/workflows/python-app.yml:49]: There is already a Test using unittest block on the staging branch. Did you checkout this branch from a recent staging commit? I think you need to rebase this branch

File: .github/workflows/python-app.yml [hunk 3]
```diff
@@ -36,6 +36,26 @@ jobs:
...
+
+    - name: Run Migrations
+      env:
+        TEST_TOKEN: ${{ secrets.TEST_TOKEN }}
+        DATABASE_URL: "postgresql://ypfrxmvtyuyudw:e0663560718f22f8b2fb44310d5da8f6e5b95f6be52476dc5f3e97316ec78ba6@ec2-18-210-159-154.compute-1.amazonaws.com:5432/dbkbegkmfukgcp"
```
Comment (kosyfrances) [inline .github/workflows/python-app.yml]: Why do you have your database URL here and not as an env var? 🤔

**Review by kosyfrances — COMMENTED** [review]
Why do you have requirements.txt file as a binary file here when we already have one on the staging branch? 🤔 
You might need to do a rebase of this branch.

File: migrations/versions/6bbae2fceac4_.py (hunk 1 above)
Comment (seun-beta) [reply migrations/versions/6bbae2fceac4_.py:68]: It is an automatically created migration file

File: .github/workflows/python-app.yml (hunk 3 above)
Comment (seun-beta) [reply .github/workflows/python-app.yml]: I didn't have access to the settings page on the repo. Had to create a temporary db to test

File: .github/workflows/python-app.yml (hunk 2 above)
Comment (seun-beta) [reply .github/workflows/python-app.yml:49]: Yes, I did. The unittest step was after the slack notification step. Just repositioned it only.

**Comment by kosyfrances** [comment]
Why do you have another requirements.txt file in this PR? 🤔

**Pull request closed without merging**